### PR TITLE
Event cutomization

### PR DESCRIPTION
**Pull request by gabibguedes**
- Adds event modal
- Adds navbar
- Colors event according to user's color and booking status
- Booking accept and reject logic

### CODE REVIEW

File: src/models/event.ts [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+
 export interface Event {
+
     start: Date
     end: Date
     title: string
+    color: string
+    modalTitle: string
+    modalDescription: string
+    type: 'checkin' | 'booking'
```
Comment (gabibguedes) [inline src/models/event.ts:11]: @GabrielDVpereira Do you remember how to do so the local and the status depends on the type? As we've seen the other day?

File: src/components/EventModal.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                          (<><b>Destino:</b> {event?.local}<br/></>)
+                          : (<><b>Status:</b> {event?.status}<br/></>)
+                        }
+                        {event?.modalDescription}
+                      </p>
+                    </div>
+                    {/*footer*/}
+                    <div className="flex items-center justify-end p-6 border-t border-solid border-slate-200 rounded-b">
+                      {event?.type === 'booking' && isAdmin && event.status === 'Pendente' && (
+                        // TODO: create function to actualy aprove and reject bookings
```
Comment (gabibguedes) [inline src/components/EventModal.tsx]: I've just left this `TODO` tag here because I think these functions for updating the booking will be a little difficult, so I think it is better to leave it to another PR.

Comment (gabibguedes) [reply src/components/EventModal.tsx]: I've added this logic here already. I originally did it on another branch, but this PR wasn't merged so I just added in this PR

**Review by GabrielDVpereira — APPROVED** [review]
